packages/lesswrong/server/fmCrosspost/resolvers.ts

```diff
@@ -27,15 +27,27 @@ export const makeCrossSiteRequest = async <RouteName extends ValidatedPostRouteN
 ): Promise<PostResponseTypes<RouteName>> => {
   const route: ValidatedPostRoutes[RouteName] = validatedPostRoutes[routeName];
   const apiUrl = makeApiUrl(route.path);
-  const result = await fetch(apiUrl, {
-    method: "POST",
-    headers: {
-      "Content-Type": "application/json",
-      "User-Agent": crosspostUserAgent,
-    },
-    body: JSON.stringify(body),
-  });
-  const json = await result.json();
+  let result: Response;
+  try {
+    result = await fetch(apiUrl, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        "User-Agent": crosspostUserAgent,
+      },
+      body: JSON.stringify(body),
+    });
+  } catch (e) {
+    if (e.cause.code === 'ECONNREFUSED' && e.cause.port === 4000) {
+      // We're testing locally, and the x-post server isn't running
+      // Slime
```

Comment:
Slime?

Reply:
Maybe I'm not using the term right. In TDD, when you're constructing your test suite, you want your tests to force your function into its behavior. So you often will wright:

```py
def sqrt(num):
    return 2

assert sqrt(4) == 2
```

And then go on to write the next, more complicated test, that will force you to right the next level of correct behavior. The `return 2` is called sliming.